docs/devel/pull-requests.md

```diff
@@ -86,7 +86,7 @@ This section applies only to pull requests on the master branch.
 
 The only exception to these rules is when a PR is not a cherry-pick and is
 targeted directly to the non-master branch.  In this case, a `release-note-*`
-label is required for that non-master PR.
+label (that is not release-note-none) is required for that non-master PR.
```

Comment:
Didn't we agree that there were cherry pick PRs (e.g. to fix test infra) that we'd take onto the release branch without a release note?